oi/type_graph/DrgnParser.cpp

```diff
@@ -71,6 +71,10 @@ Primitive::Kind primitiveFloatKind(struct drgn_type* type) {
   }
 }
 
+void warnForDrgnError(struct drgn_type* type,
```

Comment:
It would need passing through but could we consider the strict mode I added recently here? If in strict mode throw, else warn.

Reply:
I'm not sure about this - it's down to bad DWARF so there's no action that users or OI developers could take to resolve this warning/error. That means they'd have to disable strict mode in order to progress, but now that strict mode's disabled they might miss any other errors that appear later.